lib/basechurch/engine.rb

```diff
@@ -8,6 +8,7 @@
 require 'friendly_id'
 require 'validate_url'
 require 'rails-api'
+require 'settingslogic'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.